src/plots/plot.ts

```diff
@@ -465,7 +484,9 @@ export class Plot extends Component {
           index: datasetIndex,
           dataset: dataset,
           position: position,
-          selection: drawer.selectionForIndex(validDatumIndex),
+          drawer: drawer,
+          validDatumIndex: validDatumIndex,
+          selection: null,
```

Comment:
`entity.drawer.selectionForIndex(entity.validDatumIndex)`?  This one call has caching built in, so I don't see why `_lightweightEntities` doesn't